src/serde_obkv/value/mod.rs

```diff
@@ -480,7 +496,7 @@ impl Value {
                 meta.len() + util::encoded_length_vi64(f.to_bits() as i64)
             }
             Value::Date(d, ref meta) => meta.len() + util::encoded_length_vi32(d),
-            Value::Time(d, ref meta) => meta.len() + util::encoded_length_vi64(d),
+            Value::Time(d, ref meta) => meta.len() + util::encoded_length_vi64(d * 1000),
```

Comment:
Why use `d*1000` as the encoded value? If it defined by the protocol, how about keeping `d*1000` as the inner value of `Value`? And after that, there is no need to do `d*1000` and `d/1000` in the encoding and decoding procedure.

Reply:
The OB Server stores _datetime_ and _timestamp_ types using microseconds. Other clients retain data in milliseconds because obtaining microseconds is not straightforward.
However, I have discovered that microseconds can be easily retrieved using the _chrono_. Therefore, as you suggested, we store our data in microseconds in Value.

Reply:
If so, I guess the comment about the `Value::Time` should some updates:
```
Time(i64, ObjMeta), //datetime, timestamp, in millseconds
```

😄

Reply:
Fixed